src/schema/v2/home/home_page_artist_module.ts

```diff
@@ -46,21 +51,66 @@ export const HomePageArtistModuleTypes: {
       return suggestedSimilarArtistsLoader({
         exclude_followed_artists: true,
         exclude_artists_without_forsale_artworks: true,
-      }).then(({ body }) => map(body, "artist"))
+      }).then(({ body }) =>
+        body.map(({ artist, sim_artist }) => ({
```

Comment:
Gravity's response here is to include an object with both an `artist` and a `sim_artist` (the similarity basis for the match). We were throwing it away with the previous implementation (`map(body, "artist")`). This includes it.